manager/src/main/scala/org/openinfralabs/caerus/cache/manager/SemanticCacheManager.scala

```diff
@@ -58,10 +59,17 @@ class SemanticCacheManager(execCtx: ExecutionContext, conf: Config) extends Lazy
       Mode(3)
     }
   }
+  // print out tier configuration for mode3
+  if (operationMode == Mode.FULLY_AUTOMATIC && tiers.size >1) {
+    logger.info("Tiers info for Mode 3")
+    for((tier, path) <- tiers) logger.info("Tier: %s, Path: %s".format(tier, path))
```

Comment:
You should be able to do this to transform a HashMap to a string for printing:
tiers.map(tierPair => "Tier: %s, Path: %s".format(tierPair._1, tierPair._2)).mkString("\n")

It is more scala-like.

Reply:
ok, thanks